### DISCUSSION TITLE
The expression in the comment does not take effect

### DISCUSSION THREAD

**Discussion by chengjiazhao** [jxlsteam/jxls#140] [Q&A] [answered]
Hello，
I found that some of the expressions added in the comments of the Excel template take effect, and some of them do not take effect. I guess it is caused by too many excel columns.

![image](https://user-images.githubusercontent.com/42095720/126306580-be019e3b-bbaa-413b-9d5a-3a2e5444fa3a.png)
The expression to the right of the red line does not take effect

**Comment by SoltauFintel** [comment, marked as answer]
see #138
